packages/dropdowns/src/styled/items/StyledItem.ts

```diff
@@ -26,13 +28,22 @@ export const getItemPaddingVertical = (props: IStyledItemProps & ThemeProps<Defa
 };
 
 const getColorStyles = (props: IStyledItemProps & ThemeProps<DefaultTheme>) => {
+  let foregroundColor;
+  let backgroundColor;
+
+  if (props.disabled) {
+    foregroundColor = getColor('neutralHue', 400, props.theme);
+  } else if (props.isDanger) {
+    foregroundColor = getColor('dangerHue', 600, props.theme);
+    backgroundColor = props.isFocused ? rgba(foregroundColor!, 0.06) : 'inherit';
```

Comment:
Plz note that this opacity value doesn't match the `0.08` used in other parts of the system. i.e. `Button`

Reply:
Consulted with @bethanybeech and decided to stick with `0.08` for `isDanger` background-color.